Makefile

```diff
@@ -1,2 +1,2 @@
-install:
-	echo Nothing to be done for install
+.DEFAULT::
```

Comment:
.DEFAULT doesn't work if no target is given i.e typing just "make", if is a default rule but not a default target. So we should still have a target in the Makefile as well, proably "install". Also did we think it would be useful to have a clean that deleted .pyc ?

Reply:
Fixed